src/WInterop.Desktop/Windows/Native/Imports.UxTheme.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Jeremy W. Kuhne. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+using WInterop.Errors;
+using WInterop.Support;
+
+namespace WInterop.Windows.Native
+{
+    /// <summary>
+    ///  Direct usage of Imports isn't recommended. Use the wrappers that do the heavy lifting for you.
+    /// </summary>
+    public static partial class Imports
+    {
+        // https://docs.microsoft.com/windows/win32/api/uxtheme/nf-uxtheme-getcurrentthemename
+        [DllImport(Libraries.UxTheme, ExactSpelling = true)]
+        private static unsafe extern HResult GetCurrentThemeName(
+            char* pszThemeFileName,
+            int cchMaxNameChars,
+            char* pszColorBuff,
+            int cchMaxColorChars,
+            char* pszSizeBuff,
+            int cchMaxSizeChars);
+
+        public static unsafe HResult GetCurrentThemeName(
```

Comment:
I don't do overloads in "Imports" anywhere else. Use the import directly in the helper method.